### ISSUE TITLE
shopify.order.list is broken

### ISSUE THREAD

**Issue by ryanbelt** [MONEI/Shopify-api-node#145]
i am using the newest 2.9.0, 

```
let Shopify = new ShopifyAPI({
        shopName: shopifyAuth.my_shop,
        apiKey: shopifyAuth.api_key,
        password: shopifyAuth.password
    });
```
and this code is broken 
```
Shopify.order.list({created_at_min: minCreateTime, created_at_max: maxCreateTime, limit: shopifySettings.MAX_ORDER_LIST})
```
with this error
```
"err": {
        "message": "this is not a typed array.",
        "name": "TypeError",
        "stack": "TypeError: this is not a typed array.\n    at Function.from (native)\n    at Shopify.request (/var/task/node_modules/shopify-api-node/index.js:109:25)\n    at Order.list (/var/task/node_modules/shopify-api-node/mixins/base.js:72:25)\n    at /var/task/index.js:205:25\n    at tryCatcher (/var/task/node_modules/bluebird/js/release/util.js:16:23)\n    at Promise._settlePromiseFromHandler (/var/task/node_modules/bluebird/js/release/promise.js:512:31)\n    at Promise._settlePromise (/var/task/node_modules/bluebird/js/release/promise.js:569:18)\n    at Promise._settlePromise0 (/var/task/node_modules/bluebird/js/release/promise.js:614:10)\n    at Promise._settlePromises (/var/task/node_modules/bluebird/js/release/promise.js:693:18)\n    at Async._drainQueue (/var/task/node_modules/bluebird/js/release/async.js:133:16)\n    at Async._drainQueues (/var/task/node_modules/bluebird/js/release/async.js:143:10)\n    at Immediate.Async.drainQueues [as _onImmediate] (/var/task/node_modules/bluebird/js/release/async.js:17:14)\n    at processImmediate [as _immediateCallback] (timers.js:383:17)"
    },
```

But after I change back to 2.1.0 it won't show up anymore

**Comment by lpinca** [comment]
What version of Node.js are you using? You need version 4.5.0 or greater.

**Comment by lpinca** [comment]
This should have been fixed by aad804972264723f94ba6edfe731e8c036182b94. Please comment if this is not the case.
